trafficcloud/pm.py

```diff
@@ -26,65 +25,53 @@ class ProjectWizard():
-        self.project_name = str(identifier)
-        directory_names = ["homography", ".temp/test/test_object/", ".temp/test/test_feature/", "run", "results"]
-        pr_path = os.path.join(self.DEFAULT_PROJECT_DIR, self.project_name)
+        directory_names = ["homography", os.path.join(".temp", "test", "test_object"), os.path.join(".temp", "test", "test_feature"), "run", "results"]
+        project_path = get_project_path(identifier)
 
-        if not os.path.exists(pr_path):
+        if not os.path.exists(project_path):
             # Create proj dirs
-            self.PROJECT_PATH = pr_path
             for new_dir in directory_names:
-                os.makedirs(os.path.join(pr_path, new_dir))
+                os.makedirs(os.path.join(project_path, new_dir))
 
             # Write files from client
             for key,value in self.dict_files.iteritems():
                 if key[:5] == 'video':
-                    self.videopath = os.path.join(pr_path, key)
-                with open(os.path.join(pr_path, key), 'wb') as fh:
+                    self.videopath = os.path.join(project_path, key)
+                with open(os.path.join(project_path, key), 'wb') as fh:
                     fh.write(value)
 
 
-            self._write_to_project_config()
-            copy(os.path.join(os.path.dirname(os.path.realpath(__file__)), "default/tracking.cfg"), os.path.join(pr_path, "tracking.cfg"))
+            self._write_to_project_config(identifier)
+
+            default_files_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "default")
 
-            copy(os.path.join(os.path.dirname(os.path.realpath(__file__)), "default/classifier.cfg"), os.path.join(pr_path, "classifier.cfg"))
-            with open(os.path.join(pr_path, 'tracking.cfg') ,'r+') as trkcfg:
+            copy(os.path.join(default_files_dir, "tracking.cfg"), os.path.join(project_path, "tracking.cfg"))
+            copy(os.path.join(default_files_dir, "classifier.cfg"), os.path.join(project_path, "classifier.cfg"))
+
+            with open(os.path.join(project_path, 'tracking.cfg') ,'r+') as trkcfg:
                 old = trkcfg.read()
                 trkcfg.seek(0)
-                newline = 'classifier-filename = ../project_dir/{}/classifier.cfg'.format(self.project_name)
+                newline = 'classifier-filename = '+os.path.join(project_path, 'classifier.cfg')
                 trkcfg.write(newline+old)
-            with open(os.path.join(pr_path, 'classifier.cfg'),'r+') as newcfg:
+
+            with open(os.path.join(project_path, 'classifier.cfg'),'r+') as newcfg:
                 old = newcfg.read()
                 newcfg.seek(0)
-                nline1 = 'pbv-svm-filename = ../project_dir/{}/modelPBV.xml\n'.format(self.project_name)
-                nline2 = 'bv-svm-filename = ../project_dir/{}/modelBV.xml\n'.format(self.project_name)
+                nline1 = 'pbv-svm-filename = '+os.path.join(default_files_dir, 'modelPBV.xml') + '\n'
```

Comment:
don't we have methods to write config files?  (Instead of writing it hardcoded like this?) Anyway... Not a big deal